tests/spread/integration/zip/task.yaml

```diff
@@ -0,0 +1,20 @@
+summary: Integration tests for zip
+
+execute: |
+  rootfs="$(install-slices zip_bins)"
+  
+  chroot "${rootfs}/" zip -h2
+  chroot "${rootfs}/" zipcloak -h
+  chroot "${rootfs}/" zipnote -h
+  chroot "${rootfs}/" zipsplit -h
+
+  echo "Hello, World" > "${rootfs}/testfile.txt"
+  chroot "${rootfs}" zip testfile.zip testfile.txt
+  chroot "${rootfs}" zipnote testfile.zip > "${rootfs}/foo.tmp"
+  echo -e "@ This is a test comment" >> "${rootfs}/foo.tmp"
+  chroot "${rootfs}" zipnote -w testfile.zip < "${rootfs}/foo.tmp"
+  test "$(chroot foo zipnote testfile.zip | tail -n1)" = "\@ this is a test comment"
+  chroot "${rootfs}" zipsplit testfile.zip
+  chroott "${rootfs}" unzip testfil1.zip testfile.txt > "${rootfs}/testfil1.txt"
+  test '$(cat "${rootfs}/testfil1.txt")' = '$(cat "${rootfs}/testfile.txt")'
+  chroot "${rootfs}" zipgrep "Hello" testfile.zip
```

Comment:
Oops, I was mistaken earlier about zipgrep: since it is part of the unzip package, we should put it's integration test into `unzip/task.yaml`.

Reply:
Can you move the `zipgrep` test over into `unzip/task.yaml` since it is part of the `unzip` package and not `zip` package?